selectors.yml

```diff
@@ -17,6 +17,9 @@ selectors:
   1442:
     selector: 11059667695644972511
     name: "ethereum-testnet-goerli-polygon-zkevm-1"
+  2221:
+    selector: 2110537777356199208
+    name: "kava-testnet"
```

Comment:
Afaik Kava is an L1 chain that is both evm and cosmos compatible. Any thoughts about naming @andrevmatos ?

Reply:
Compatibility shouldn't matter much here for naming; if it's a L1 (afaics as well), then the naming here should be right